src/components/menus/content-contextual-menu.js

```diff
@@ -94,14 +94,12 @@ const ContentContextualMenu = (props) => {
 
     const duplicateItem = () => {
         if (activeElement) {
-            const duplicateSuffix = '(1)';
-            const newItemName = activeElement.elementName + duplicateSuffix;
-            elementExists(
+            getNameCandidate(
                 selectedDirectory.elementUuid,
-                newItemName,
+                activeElement.elementName,
                 activeElement.type
-            ).then((data) => {
-                if (!data) {
+            ).then((newItemName) => {
```

Comment:
Post aprove remark : maybe implement a catch in order to display unexpected errors ?